src/other/index.jl

```diff
@@ -195,11 +192,9 @@ function add_names(ind::Index, add_ind::Index; makeunique::Bool=false)
     end
     if length(dups) > 0
         if !makeunique
-            Base.depwarn("Duplicate variable names are deprecated: pass makeunique=true to add a suffix automatically.", :add_names)
-            # TODO: uncomment the lines below after deprecation period
-            # msg = """Duplicate variable names: $(u[dups]).
-            #          Pass makeunique=true to make them unique using a suffix automatically."""
-            # throw(ArgumentError(msg))
+            msg = "Duplicate variable names: $(u[dups]). Pass makeunique=true" *
```

Comment:
AFAICT this will print as `Symbol[:col]`. I think you can get something really nice using `$(join(string.(':', u[dups]), ", ", " and "))`. Same in other places (including in untouched code above).

Reply:
OK - fixed. I have also made it to avoid printing the same name of an offending column more than once.